lib/frizz/site.rb

```diff
@@ -3,16 +3,16 @@ class Site
     def initialize(host, options={})
       @options = { from: "build" }.merge options
```

Comment:
Should we default `prefer_gzip` to `false` here? Currently it'll just be `nil` if not provided by the user — which while `nil` is falsey, it's less explicit than we could make it by defaulting to `false`